app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -93,6 +94,11 @@ class PresetsFragment : BaseFragment<FragmentPresetsBinding>() {
             findNavController().navigate(R.id.action_presetsFragment_to_settingsActivity)
         }
 
+        binding.emptyAddPhraseButton.action = {
+            val action = PresetsFragmentDirections.actionPresetsFragmentToKeyboardFragment()
```

Comment:
We'll probably want to pass the current category to the keyboard fragment so we can set that category to checked on [this screen](https://www.figma.com/file/mNrwUygVhTmuWnKNHHkenR/Vocable-New?node-id=136%3A466). We can do that in another PR though since that flow will come later.

Reply:
Yeah my plan is to do this with the PR that adds the screen with the switches.